src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4601,10 +4601,10 @@ paths:
           required: true
           schema:
             type: string
-        - in: path
+        - in: query
           description: User specified text to search concepts for.
           name: searchText
-          required: true
+          required: false
```

Comment:
instead of having to pass in the empty string for the initial render on the UI, nothing needs to be passed in

Reply:
I think `required: false` is the default so it can now be omitted.